src/app/community/community.component.ts

```diff
@@ -11,8 +11,12 @@ export class CommunityComponent implements OnInit {
   communities = [];
   filter = '';
   selectedValue = '';
+  nationName = '';
+
   constructor(
-    private couchService: CouchService
+    private couchService: CouchService,
+    private url:LocationStrategy,
```

Comment:
this code is giving problem 
![screen shot 2017-11-14 at 8 56 00 am](https://user-images.githubusercontent.com/24647983/32761131-82af155e-c8e9-11e7-8822-26cc7c27ad0f.png)

Reply:
thanks @razu9861 :+1: